### PR TITLE
add inline keyboard for help command

### PR DESCRIPTION
**Pull request by ayrat555**

### CODE REVIEW

File: src/bot/update_handler.rs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            UpdateContent::ChannelPost(channel_post) => channel_post,
+            _ => return,
+        };
+
+        if let Some(owner_id) = Self::owner_telegram_id() {
+            if message.from.is_none() {
+                return;
+            }
+
+            if message.from.as_ref().unwrap().id as i64 != owner_id {
```
Comment (pxp9) [inline src/bot/update_handler.rs:53]: `?` operator ?

File: src/bot/update_handler.rs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            }
+        }
+
+        let text = message.text.clone();
+
+        if text.is_none() {
+            return;
+        }
+
+        let command = BotCommand::from_str(&text.unwrap()).unwrap();
```
Comment (pxp9) [inline src/bot/update_handler.rs:64]: `?` operator ?

File: src/bot/update_handler.rs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            _ => return,
+        };
+
+        let text = query.data.clone();
+
+        if text.is_none() || query.message.is_none() {
+            return;
+        }
+
+        let command = BotCommand::from_str(&text.unwrap()).unwrap();
```
Comment (pxp9) [inline src/bot/update_handler.rs:85]: `?` operator ?

File: src/bot/update_handler.rs [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+        CommandProcessor::builder()
+            .message(query.message.unwrap())
```
Comment (pxp9) [inline src/bot/update_handler.rs:88]: `?` operator ?

**Review by pxp9 — APPROVED** [review]
Looks good, just a few comments.

File: src/bot/update_handler.rs (hunk 4 above)
Comment (ayrat555) [reply src/bot/update_handler.rs:88]: there is a check above that checks if message is none

File: src/bot/update_handler.rs (hunk 3 above)
Comment (ayrat555) [reply src/bot/update_handler.rs:85]: any string can be converted to BotCommand without errors

File: src/bot/update_handler.rs (hunk 1 above)
Comment (ayrat555) [reply src/bot/update_handler.rs:53]: there is a check above that checks if from is none

File: src/bot/commands/help.rs [hunk 5]
```diff
@@ -50,10 +144,68 @@ impl Help {
     pub fn command() -> &'static str {
         COMMAND
     }
+
+    pub fn help_keyboard_params(&self) -> SendMessageParams {
+        let mut buttons: Vec<Vec<InlineKeyboardButton>> = Vec::new();
+
```
[the remark this answers is not in the document]
Comment (ayrat555) [reply src/bot/commands/help.rs]: done

File: src/bot/commands.rs [hunk 6]
```diff
@@ -210,19 +225,34 @@ pub trait Command {
...
             self.api()
                 .reply_with_text_message(message.chat.id, text, Some(message.message_id))
         {
-            error!("Failed to reply to update {:?} {:?}", error, message);
+            error!("Failed to reply to a message {:?} {:?}", error, message);
         }
     }
 
+    fn send_message(&self, send_message_params: SendMessageParams) {
+        if let Err(error) = self.api().send_message_with_params(&send_message_params) {
```
[the remark this answers is not in the document]
Comment (ayrat555) [reply src/bot/commands.rs:244]: for now errors are just logged, the result of requests is not used

**Pull request merged**